.github/workflows/publish.yml

```diff
@@ -83,7 +83,7 @@ jobs:
       - name: Set environment information for schedule
         if: ${{ steps.check_event.outputs.event_type == 'schedule' }}
         run: |
-          LATEST_VERSION=$( gh release view --json tagName --jq '.["tagName"]' )      
+          LATEST_VERSION=$(curl --silent -m 10 --connect-timeout 5 https://api.github.com/repos/complytime/trestle-bot/releases/latest | jq .tag_name)      
```

Comment:
The only difference I see is that the new command return the tag within `"`
```
gh release view --json tagName --jq '.["tagName"]'
v0.11.0

curl --silent -m 10 --connect-timeout 5 https://api.github.com/repos/complytime/trestle-bot/releases/latest | jq .tag_name
"v0.11.0"
```

A minor update in the last command to include `-r` could help if the quotes may bring issues:
```
curl --silent -m 10 --connect-timeout 5 https://api.github.com/repos/complytime/trestle-bot/releases/latest | jq -r .tag_name
v0.11.0
```